src/appleseed-max-impl/appleseedrenderer/appleseedrendererparamdlg.cpp

```diff
@@ -563,6 +566,9 @@ namespace
         virtual void init(HWND hwnd) override
         {
             m_static_project_filepath = GetDlgItem(hwnd, IDC_STATIC_PROJECT_FILEPATH);
```

Comment:
`m_static_project_filepath` and `m_text_project_filepath` should probably stay together.

Reply:
I'll move it here and change declaration order too...